backend/clap/proto/data_store.proto

```diff
@@ -7,113 +7,129 @@ option java_outer_classname = "DataStoreProto";
+  int32 submission_id = 1;
 }
 
 message SubmissionsResponse {
-    repeated Submission submissions = 1;
+  repeated Submission submissions = 1;
 }
 
 message Submission {
-    int32  submission_id = 1;
-    int32  year          = 2;
-    string name          = 3;
-    int32  duration_days = 4;
+  int32  submission_id = 1;
+  int32  year = 2;
+  string name = 3;
+  int32  duration_days = 4;
 
-    repeated Assessor assessors = 5;
-    repeated Rating   ratings   = 6;
+  repeated Assessor assessors = 5;
+  repeated Rating   ratings = 6;
 }
 
 message Assessor {
-    string first_name  = 1;
-    string last_name   = 2;
-    int32  assessor_id = 3;
+  string first_name = 1;
+  string last_name = 2;
+  int32  assessor_id = 3;
 }
 
 message Rating {
-    int32      rating_id   = 1;
-    bool       is_draft    = 2;
-    int32      assessor_id = 3;
-    RatingType type        = 4;
+  int32      rating_id = 1;
+  bool       is_draft = 2;
+  int32      assessor_id = 3;
+  RatingType type = 4;
 }
 
 message DetailsSubmissionResponse {
-    int32        team_size   = 1;
-    string       finish_date = 2;
-    ProjectState status      = 3;
-    string       budget      = 4;
-    string       description = 5;
-    AppReport    report      = 6;
+  int32        team_size = 1;
+  string       finish_date = 2;
+  ProjectState status = 3;
+  string       budget = 4;
+  string       description = 5;
+  AppReport    report = 6;
 }
 
 message AppReport {
-    bool is_draft                 = 1;
-    string report_submission_date = 2;
-    string project_goals          = 3;
-    string organisation_structure = 4;
-    string division_of_work       = 5;
-    string project_schedule       = 6;
-    string attatchments           = 7;
+  bool is_draft = 1;
+  string report_submission_date = 2;
+  string project_goals = 3;
+  string organisation_structure = 4;
+  string division_of_work = 5;
+  string project_schedule = 6;
+  string attatchments = 7;
 }
 
 message Criterion {
-    int32 criterion_id = 1;
-    string name        = 2;
-    string description = 3;
-    string area        = 4;
-    string criteria    = 5;
-    string subcriteria = 6;
+  int32 criterion_id = 1;
+  string name = 2;
+  string description = 3;
+  string area = 4;
+  string criteria = 5;
+  string subcriteria = 6;
 }
 
 message PartialRating {
-    int32  partial_rating_id = 1;
-    int32  criterion_id      = 2;
-    int32  points            = 3;
-    string justification     = 4;
+  int32  partial_rating_id = 1;
+  int32  criterion_id = 2;
+  int32  points = 3;
+  string justification = 4;
 
-    string modified          = 5;
-    int32  modified_by       = 6;
+  string modified = 5;
+  int32  modified_by = 6;
 }
 
 message AssessorRatings {
-    int32 assessor_id = 1;
-    string first_name = 2;
-    string last_name  = 3;
-    repeated PartialRating partial_ratings = 4;
+  int32 assessor_id = 1;
+  string first_name = 2;
+  string last_name = 3;
+  repeated PartialRating partial_ratings = 4;
```

Comment:
Could you please add the `is_draft` here and expose it through the Clap API? @Veermove as it requires your effort too.